src/preset-themes/jsonSchema/jsonSchema.js

```diff
@@ -0,0 +1,227 @@
+import { makeLog } from "../../lib/utils";
+
+const log = makeLog("background");
+
+var Ajv = require("ajv");
+var ajv = new Ajv({
+    allErrors: true
+});
+
+var schema = {
+    "properties": {
+        "title": {
+            "type": "string"
+        },
+        "colors": {
+            "type": "object",
+            "properties": {
+                "toolbar": {
+                    "type": "object",
+                    "properties": {
+                        "h": {
+                            "type": "number",
```

Comment:
small nit: "h", "s", and "l" should be integers, and I think only "a" supports a floating point value, per:

https://github.com/mozilla/Themer/blob/38933c93a6c304a70b095dfd737b3b2c8ce871d6/src/preset-themes/001.json#L25-L30